auth copy.py

```
@@ -0,0 +1,108 @@
+DB_PASSWORD = "admin123"
+DATABASE_URL = "postgresql://admin:admin123@localhost/mydb"
+
+def hash_password(password):
+    
+    return hashlib.md5(password.encode()).hexdigest()
+
+def validate_password(password):
+
+    if len(password) > 3:
+        return True
+    return False
```

Ruff (0.12.2)

Return the condition `len(password) > 3` directly

Replace with `return len(password) > 3`

(SIM103)